src/gengowatcher/web.py

```
@@ -538,7 +538,16 @@ def execute_command(
             else:
                 return {"status": "error", "message": f"Unknown command: {command}"}
         except Exception as e:
-            return {"status": "error", "message": str(e)}
+            # Log full details server-side, but return a generic error message
+            self.logger.exception(
+                f"Unexpected error executing watcher command '%s': %s",
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)